extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.css

```diff
@@ -40,20 +45,45 @@
   height: 120px !important;
 }
 
+.i-amphtml-amp-story-audio-sticker-container * {
+  pointer-events: none !important;
+}
+
 amp-story-audio-sticker {
+  position: relative !important;
+  width: 100% !important;
   max-width: 100% !important; /* Set the maximum width of the image to 100% to ensure it fits within the container */
   max-height: 100% !important; /* Set the maximum height of the image to 100% to ensure it fits within the container */
-  margin: auto !important; /* Center the image horizontally and vertically within the container */
-}
-
-amp-story-audio-sticker * {
-  pointer-events: none !important;
 }
 
 amp-story[muted] amp-story-audio-sticker {
   pointer-events: auto !important;
 }
 
-amp-story:not([muted]) .i-amphtml-amp-story-audio-sticker-component {
-  display: none !important;
+/*
+  Hide the sticker after the story gets unmuted:
+  1) if the sticker is on the active page, hide it after showing posttap sticker for 4 seconds.
+  2) if the sticker is not on the active page, hide it immediately since it is in the background.
+*/
+amp-story:not([muted]) amp-story-audio-sticker.on-active-page {
+  transition: opacity 0.5s ease 4s !important;
```

Comment:
You can just change `transition-delay: 4s` instead of re-defining all the values

Reply:
Oh, I misread the selector above that has `.i-amphtml-amp-story-audio-sticker-tap-hint {transition: opacity. 0.5s ease; }`. Closing this